composer.json

```diff
@@ -18,6 +18,7 @@
 		"wmde/fundraising-donations": "@dev",
 		"wmde/fundraising-memberships": "@dev",
 		"wmde/fundraising-payments": "@dev",
+		"wmde/fundraising-subscriptions": "fix-validations-dev",
```

Comment:
Must be changed to `@dev` when https://github.com/wmde/fundraising-subscriptions/pull/3 is merged.